.github/workflows/test.yml

```diff
@@ -87,6 +89,14 @@ jobs:
     needs: check_changes
     if: github.event_name != 'pull_request' || contains(needs.check_changes.outputs.path_filters, 'cbbr')
     uses: ./.github/workflows/cbbr_test.yml
+    secrets: inherit
+  cpdb:
+    needs: check_changes
+    if: github.event_name != 'pull_request' || contains(needs.check_changes.outputs.path_filters, 'cpdb')
+    uses: ./.github/workflows/cpdb_build.yml
+    with:
+      run_export: true
+    secrets: inherit
```

Comment:
did this rather than creating a `cpdb_test.yml` to run a build. I figured we want less workflow files and probably want to discourage product-level pytests

Reply:
Why do we want to discourage product-level pytests?

Reply:
sorry yea that does sound drastic. i guess if all the `products` folders are so "transforms for this product", writing more python doesn't feel as likely as writing more SQL

no doubt we have python code in those folders at the moment though